docs/overview/changelog.md

```diff
@@ -17,8 +17,10 @@ _Released 2023-11-16_
 - Added support for following a topic, with configurable notification
   settings for followed topics and flexible configuration options to
   automatically follow topics when sending a message or otherwise
-  interacting with it. New `Shift + N` keyboard shortcut navigates to
-  the next unread followed topic.
+  interacting with it, or when mentioned. New `Shift + N` keyboard
+  shortcut navigates to the next unread followed topic.
+- Added support for limiting guest users' access to everyone in an
+  organization.
```

Comment:
How about:

-  Added support for limiting user list access for guests.

With the current wording, the scope for "everyone" seems ambiguous.